roles/openshift_hosted/meta/main.yml

```diff
@@ -12,5 +12,23 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
-- openshift_common
-- openshift_hosted_facts
+- role: openshift_common
```

Comment:
I'm wondering if it makes sense to put a dependency on openshift_cli, since that is really a requirement here.